samples/drivers/led/pwm/sample.yaml

```diff
@@ -17,6 +17,6 @@ tests:
         - "Turned on"
         - "Turned off"
         - "Increasing brightness gradually"
-        - "Blinking on: 0.1 sec, off: 0.1 sec"
-        - "(Blinking on: 1 sec, off: 1 sec|Cycle period not supported)"
```

Comment:
You can still have `Cycle period not supported` logged. So you have to keep that part.

Reply:
OK, restored